user_settings.sql

```diff
@@ -0,0 +1,79 @@
+-- This is just a test (in draft) file so that Arit and I can work together using revision control
+-- We run the contents of this file in the rails dbconsole
+-- It will not be committed to the codebase, it will be moved into a data update script and this file will be deleted :)
+
+BEGIN TRANSACTION;
+WITH settings_data AS (
+  SELECT
```

Comment:
@msarit based on this comment:
> While working on this with Rhymes, we discovered that some settings that have null: false defined on them may currently be null in the database. For these, I utilized the COALESCE function, which returns the first non-null argument. Wherever I used this function in the scripts above, I passed in the default value for that setting as the second argument.

Have we confirmed that it was only needed on `display_email_on_profile` in the `users_settings` table? Are there any others that require us to use the COALESCE? Should we use this function for all columns that are `null: false`?

Reply:
I did this check for `users_notification_settings`; I will double check for `users_settings`.